doc/README.md

```diff
@@ -94,6 +94,8 @@ Some common pprof options are:
   *regex*.
 * **-ignore= _regex_:** Do not include samples that include a report entry matching
   *regex*.
+* **-show\_from= _regex_:** Do not show entries above the first frame that 
```

Comment:
Maybe say "above the first one" as it doesn't look like the text here uses term "frame".

Reply:
done.